cmake/developer_package/frontends/frontends.cmake

```diff
@@ -139,14 +139,15 @@ macro(ov_add_frontend)
     file(GLOB proto_files ${frontend_root_dir}/src/proto/*.proto)
 
     foreach(INFILE IN LISTS proto_files)
+        set(GOOGLE_DIR ${PROJECT_SOURCE_DIR}/thirdparty/protobuf/protobuf/src)
```

Comment:
looks like a hack, do we need to use internal protobuf API?

Reply:
not sure I got your point. it is path to a google's *.proto files, which are included to original tf proto files...

Reply:
got it, thanks

OpenVINO has an ability to be compiled with system protobuf (`cmake -DENABLE_SYSTEM_PROTOBUF`). I suppose in this case we don't have those files.. maybe it's better to copy them to our repo?

Reply:
Added to source... Please check copyrights later.